crypto/asn1/a_time.c

```diff
@@ -124,14 +124,14 @@ int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *d)
             i++;
             break;
         }
-        if (!isdigit(a[o]))
+        if (!ossl_isdigit(a[o]))
             goto err;
         n = a[o] - '0';
```

Comment:
Again, the use if ossl_isdigit() indicates that the data values are in ASCII, where as the other character constants are not guaranteed to be ASCII.